extensions/markdown/package.json

```diff
@@ -73,7 +73,7 @@
 		"menus": {
 			"editor/title": [
 				{
-					"when": "resourceLangId == markdown",
+					"when": "editorLangId == markdown",
```

Comment:
I could also change this to be `resourceLangId == markdown || editorLangId == markdown`, but only showing the preview menu item for files that are viewed as markdown makes more sense  to me